specification/msi/resource-manager/Microsoft.ManagedIdentity/2015-08-31-preview/ManagedIdentity.json

```diff
@@ -0,0 +1,535 @@
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Deleted Identity.",
+            "schema": {
+              "$ref": "#/definitions/Identity"
+            }
+          },
+          "204": {
+            "description": "The specified identity does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "IdentityCreateOrUpdateParameters": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "The supported Azure region where the associated Resource Group exists."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "description": "Parameters for creating or updating identity."
+    },
+    "Identity": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The id of the created identity."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the created identity."
+        },
+        "location": {
+          "type": "string",
+          "description": "The Azure region where the identity was created."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "The properties associated with the identity.",
+          "$ref": "#definitions/IdentityProperties"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "UserAssignedIdentities"
+          ],
+          "x-ms-enum": {
+            "name": "Microsoft.ManagedIdentity/userAssignedIdentities",
+            "modelAsString": true
+          },
+          "description": "The type of resource i.e. Microsoft.ManagedIdentity/userAssignedIdentities"
+        }
+      },
+      "x-ms-azure-resource": true,
+      "description": "Describes an identity resource."
+    },
+    "IdentityProperties": {
+      "description": "The properties associated with the identity.",
+      "type": "object",
+      "title": "Identity properties.",
+      "properties": {
```

Comment:
For convention reasons, it is preferrable to provide documentation in `description` instead of `title`. In this swagger, there are many places in which title could change to description.

Reply:
Fixed title and description throughout the file